packages/react-instantsearch-core/src/connectors/__tests__/connectInfiniteHits.js

```diff
@@ -5,39 +5,35 @@ jest.mock('../../core/createConnector', () => x => x);
 describe('connectInfiniteHits', () => {
   describe('single index', () => {
     const createSingleIndexContext = () => ({
-      context: {
-        ais: {
-          mainTargetedIndex: 'index',
-        },
-      },
+      mainTargetedIndex: 'index',
     });
 
     it('provides the current hits to the component', () => {
-      const context = createSingleIndexContext();
-      const getProvidedProps = connect.getProvidedProps.bind(context);
+      const contextValue = createSingleIndexContext();
 
       const hits = [{}];
-      const props = getProvidedProps(null, null, {
+      const props = connect.getProvidedProps.call({}, { contextValue }, null, {
         results: { hits, page: 0, hitsPerPage: 2, nbPages: 3 },
       });
 
       expect(props).toEqual({ hits, hasMore: true });
     });
 
     it('accumulate hits internally', () => {
-      const context = createSingleIndexContext();
-      const getProvidedProps = connect.getProvidedProps.bind(context);
+      const contextValue = createSingleIndexContext();
 
       const hits = [{}, {}];
       const hits2 = [{}, {}];
-      const res1 = getProvidedProps(null, null, {
+      const ctx = {};
```

Comment:
`ctx` is misleading since we speak about `context` inside the props. `instance` should be ok since it's actually a fake component instance. We have to apply this change in the all file.